docs/en-US/component/notification.md

```diff
@@ -61,6 +61,18 @@ Although `message` property supports HTML strings, dynamically rendering arbitra
 
 :::
 
+## Use VNode
```

Comment:
```suggestion
## Message using functions ^(2.8.7)
```
Put the version number in the title to make it more visible.

Reply:
> Put the version number in the title to make it more visible.

The documentation here seems to have some issues, this change seems unnecessary, what do you think?

1. `Vnode` was supported from the beginning, but it was not documented

![image](https://github.com/user-attachments/assets/1174a7df-ae71-40c4-97a2-59f4e62799db)


2. This time, `function` support was added (`2.8.7`), and the return type is specifically `Vnode`, which conflicts with the title

![image](https://github.com/user-attachments/assets/0acac1fd-e21e-49ed-b74e-047bb6050172)

Reply:
> 1. `Vnode` was supported from the beginning, but it was not documented

`message` supports `vnode` type, and the document also marks this type. What is the problem you are referring to?

> 2. This time, `function` support was added (`2.8.7`), and the return type is specifically `Vnode`, which conflicts with the title

What exactly is the conflict you are talking about?

Reply:
I missed your reply, I review again, no problem.